src/sortStudents.ts

```diff
@@ -1,16 +1,79 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[], sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  // Create a new array to hold the sorted students
+  const sortedStudents = [...students];
+
+  // Define the sorting logic based on the provided sortBy parameter
+  switch (sortBy) {
+    case SortType.Name:
+      sortedStudents.sort((a, b) => a.name.localeCompare(b.name));
+      break;
+    case SortType.Surname:
+      sortedStudents.sort((a, b) => a.surname.localeCompare(b.surname));
+      break;
+    case SortType.Age:
+      sortedStudents.sort((a, b) => a.age - b.age);
+      break;
+    case SortType.Married:
+      sortedStudents.sort((a, b) => {
+        if (a.married === b.married) {
+          return 0;
+        }
+
+        if (order === 'asc') {
+          return a.married ? -1 : 1;
+        // eslint-disable-next-line no-else-return
+        } else {
+          return a.married ? 1 : -1;
+        }
```

Comment:
```suggestion
        if (order === 'asc') {
          return a.married ? -1 : 1;
        }
        
        return a.married ? 1 : -1;
```
Don't disable the eslint, but try to fix its problem. `else` is redundant here as code will not go further after return anyway